packages/@react-spectrum/datepicker/stories/TimeField.stories.tsx

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {action} from '@storybook/addon-actions';
+import {CalendarDateTime, parseTime, parseZonedDateTime, Time, toZoned} from '@internationalized/date';
+import React from 'react';
+import {storiesOf} from '@storybook/react';
+import {TimeField} from '../';
+
+const BlockDecorator = storyFn => <div>{storyFn()}</div>;
+
+storiesOf('Date and Time/TimeField', module)
+  .addDecorator(BlockDecorator)
+  .add(
+    'default',
+    () => render()
+  )
+  .add(
+    'defaultValue',
+    () => render({defaultValue: parseTime('20:24')})
+  )
+  .add(
+    'controlled value',
+    () => render({value: new Time(2, 35)})
+  )
+  .add(
+    'granularity: second',
+    () => render({granularity: 'second'})
+  )
+  .add(
+    'hourCycle: 12',
+    () => render({hourCycle: 12})
+  )
+  .add(
+    'hourCycle: 24',
+    () => render({hourCycle: 24})
+  )
+  .add(
+    'zoned',
+    () => render({defaultValue: parseZonedDateTime('2021-11-07T00:45-07:00[America/Los_Angeles]')})
```

Comment:
Flips between PST and PDT when you change the hours

Reply:
Indeed! This is intentional. It happens on daylight saving time shifts.